schemas/hash.yml

```
@@ -5,12 +5,16 @@
   type: group
   short: Hashes, usually file hashes.
   description: >
-    The hash fields represent different hash algorithms and their values.
+    The hash fields represent different bitwise hash algorithms and their values.
 
     Field names for common hashes (e.g. MD5, SHA1) are predefined. Add fields for other hashes
     by lowercasing the hash algorithm name and using underscore separators as appropriate
     (snake case, e.g. sha3_512).
 
+    Note that this fieldset should be used for common hashes that may be computed
+    over a range of generic bytes. Entity-specific hashes such as ja3 or imphash should be
+    placed in the fieldsets to which they relate (i.e. tls and pe).
```

```suggestion
    placed in the fieldsets to which they relate (tls and pe, respectively).
```

nit: minor rewording suggestion